resources/js/pusher.js

```diff
@@ -0,0 +1,70 @@
+$(document).ready(function () {
+    var pusher = new Pusher('ab562d35a669854f4888', {
+        cluster: 'ap1'
+    });
+    var user_id = $('#noti-message').data('user');;
+    var channel = pusher.subscribe('notify-for-admin' + user_id);
+    channel.bind('booking-notify', function (data) {
+        console.log('hello');
```

Comment:
giải thích đoạn code này

Reply:
Tạo một pusher để subcribe channel 'noifify-for-admin' + user_id. Trong đó user_id là id của admin , khi gửi thông báo từ phía client cũng sẽ gửi theo channel có format như thế này để admin khác nhau có thể nhận được thông báo khác nhau theo channel riêng của admin